src/pseudopeople/loader.py

```diff
@@ -24,7 +25,10 @@ def load_standard_dataset(
         if not parquet_filters:
             # pyarrow.parquet.read_table doesn't accept an empty list
             parquet_filters = None
-        data = pq.read_table(data_path, filters=parquet_filters).to_pandas()
+        data = pq.read_table(
+            str(data_path),
+            filters=parquet_filters,  # type: ignore [arg-type]
```

Comment:
I couldn't figure this one out. We pass in filters where the last item of the tuple can be a str, int, or pd.Datetime (e.g. `("date", "<=", some-datetime-object)`). But the parquet read_table() method expects a tuple[str, str, str]. The filters obviously work though, so I assume it's just a typing issue on parquet's end.

Reply:
What if we formatted the datetime object to a str?

Reply:
that seemed like a scary and fundamental change - but I guess that's what the tests are for. I'll try it

Reply:
I couldn't get this to work either. When I converted the last item of the tuple to str, I got a similar mypy error:
`error: Argument "filters" to "read_table" has incompatible type "list[tuple[str, str, str]] | None"; expected "list[tuple[str, str, str] | list[tuple[str, str, str]]] | None"  [arg-type]`

The issue seems to be that we need to convince mypy that we really will never have a nested list.

I tried casting:
```
parquet_filters = cast(
        Optional[list[tuple[str, str, str]]],
        [filter.to_tuple() for filter in filters] if filters else None,
    )
```
But that led to the exact same error.

I even tried handling the list of lists by checking and raising, but no go
```
if parquet_filters is not None:
        for item in parquet_filters:
            if isinstance(item, list):
                raise ValueError("Nested lists are not allowed in parquet_filters")
```

Reply:
If you'd like me to change that last item to a str just b/c it at least matches the type hint better, lmk and I'll test it. But for now I'm just leaving it be.